types/ModuleLoader.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+export type ModuleLoader = {
+  requireModule: () => void,
```

Comment:
you can find the API for ModuleLoader here: https://github.com/cpojer/jest/blob/master/packages/jest-cli/src/Runtime/Runtime.js#L71 – would you mind stubbing out the two methods that you need here with the full annotations?